backend/OR-Tools/OR-Tools/Solvers/OrToolsConverter.cs

```diff
@@ -0,0 +1,245 @@
+﻿using Domains.Models.Input;
+using Domains.Models.Output;
+
+using Google.OrTools.ConstraintSolver;
+
+using OR_Tools.Models;
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace OR_Tools.Solvers
+{
+    internal class OrToolsConverter
+    {
+        private static readonly int SAME_LOCATION = 0;
+        private static readonly int NO_CROSS_BETWEEN_LOCATION = 1000;
+        
+        private int ToMinutes(DateTime dt) => (dt.Hour * 60) + dt.Minute;
+
+        #region ToWorkDate
+        public WorkData ToWorkData(FileInput fileInput)
+        {
+            DateTime minDate = fileInput.Locations.Select(l => l.From).Min();
+
+            int vehiclesAmount = fileInput.Vehicles.Count();
+            int locationsAmount = fileInput.Locations.Count();
+
+            return new WorkData
+            {
+                VehiclesAmount = vehiclesAmount,
+
+                DistanceMatrix = GenerateDistancesMatrix(fileInput),
+                DurationMatrix = GenerateDurationsMatrix(fileInput),
+
+                TimeWindows = fileInput.Locations.Select(l => new[] { ToMinutes(minDate), (long)(l.To - minDate).TotalMinutes }).ToArray(),
+                ServiceTimes = fileInput.Locations.Select(l => (long)l.Service).ToArray(),
+
+                Demands = fileInput.Locations.Select(l => (long)l.Demand).ToArray(),
+                VehicleCapacities = fileInput.Vehicles.Select(v => (long)v.Capacity).ToArray(),
+
+                Starts = fileInput.Vehicles.Select(v => v.Start).ToArray(),
+                Ends = fileInput.Vehicles.Select(v => v.End).ToArray()
+            };
+        }
+
+        private long[,] GenerateLocationMatrix(int locationsAmount)
+        {
+            long[,] locations = new long[locationsAmount, locationsAmount];
+            for (int i = 0; i < locationsAmount; ++i)
+            {
+                for (int j = 0; j < locationsAmount; ++j)
+                {
+                    if (i == j) locations[i, j] = SAME_LOCATION;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4635982)